samples/Ch02_where_code_runs/fig_2_15_aspect_selector.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2020 Intel Corporation
+
+// SPDX-License-Identifier: MIT
+
+#include <sycl/sycl.hpp>
+#include <iostream>
+using namespace sycl;
```

Comment:
Have you considered not using this?

Reply:
Have we considered prefixing `sycl::` everywhere you mean, instead of `using namespace`?

Reply:
Yes. At the end, I think a reference book should show some code following best practice and this should be avoided.
In any big enough code, it would be unclear and the risk of collision would be too high.
This is probably valid for the examples from SYCL specification too.